WPMedia/fusion-news-theme-blocks
# Fix general conditions for visibility of promo components.

## Description
Updated conditionals surrounding render components involving the image (and video) tags in promotional items and in top table list promos.

## Jira Ticket
- [TMEDIA-219](https://arcpublishing.atlassian.net/browse/TMEDIA-219)

## Acceptance Criteria
Content should not change except in the case where images are the only inclusion.  In such a case, those should now be rendered.

## Test Steps
- Visit `http://localhost/pagebuilder/editor/curate?p=p0ciQAinp2VIRr&v=vFzdS81np2VIRr`
- Select the first promo item 'Extra Large Manual Promo – Arc Block'
- Open the Show Promo Elements option
  ![image](https://user-images.githubusercontent.com/2287238/118139585-7fe43400-b3d5-11eb-9530-e28fdc650e41.png)
- Deselect all but the image and verify that it still shows up without the other elements
  ![image](https://user-images.githubusercontent.com/2287238/118139706-a1452000-b3d5-11eb-9072-c20dd240f375.png)
- Repeat for each promo block type
- Also verify the 'Table Top List Block' using the same pattern.
 

1. Checkout this branch `git checkout 219-promo-block-visibility`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/extra-large-manual-promo-block,@wpmedia/extra-large-promo-block,@wpmedia/large-manual-promo-block,@wpmedia/large-promo-block,@wpmedia/medium-manual-promo-block,@wpmedia/medium-promo-block,@wpmedia/top-table-list-block`


## Effect Of Changes
### Before
Selecting only the image would show nothing
![image](https://user-images.githubusercontent.com/2287238/118140646-8e7f1b00-b3d6-11eb-8324-6841971b87de.png)

### After
Selecting only the image will now show the image
![image](https://user-images.githubusercontent.com/2287238/118140806-b79fab80-b3d6-11eb-8c7e-cb231f5b1d3a.png)

## Author Checklist
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
